sql/telemetry_derived/fission_monitoring_main_v1/query.sql

```diff
@@ -0,0 +1,11 @@
+-- creates a pre-filtered main ping dataset for monitoring
+-- and ad-hoc analyses on fission experiment
+SELECT
+  *
```

Comment:
Based on the `init.sql`, shouldn't this be something like this?
```suggestion
  DATE(submission_timestamp) AS submission_date,
  document_id,
  sample_id
```

Reply:
We want to have all the probes here, but also I don't want to specify all of them in the init.sql - therefore the `*` and `ALLOW_FIELD_ADDITION` [here](https://github.com/mozilla/telemetry-airflow/pull/1154/files#diff-da835b6a2ddbe3615ee058a2b838ccffR41)

Reply:
Ah, I see